gpu-alloc/Cargo.toml

```diff
@@ -15,9 +15,10 @@ categories = ["graphics", "memory-management", "no-std", "game-development"]
 [features]
 std = []
 default = ["std"]
+serde = ["dep:serde", "bitflags/serde"]
 
 [dependencies]
 gpu-alloc-types = { path = "../types", version = "0.2" }
 tracing = { version = "0.1.27", optional = true, features = ["attributes"], default-features = false }
-bitflags = { version = "1.2", default-features = false }
-serde = { version = "1.0", optional = true, default-features = false, features = ["derive"] }
+bitflags = { version = "2.0", default-features = false }
+serde = { version = "1.0", package = "serde", optional = true, default-features = false, features = ["derive"] }
```

Comment:
`package = "serde"` seems unnecessary.

Reply:
Ah, leftover from the previous iteration.
Fixed!